doc/source/index.rst

```diff
@@ -10,6 +10,12 @@ Tenacity
    :target: https://mergify.io
    :alt: Mergify Status
 
+For a better experience
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I'm not sure this is going to render correctly. Nobody is going to understand this relates to the icon below.
```suggestion
```